controllers/index.js

```diff
@@ -151,13 +152,30 @@ exports.home = function (aReq, aRes) {
   async.parallel(tasks, asyncComplete);
 };
 
+// Get the referer url for redirect after login/logout
+function getRedirect(aReq) {
+  var referer = aReq.get('Referer');
+  var redirect = '/';
+
+  if (referer) {
+    referer = url.parse(referer);
+    if (referer.hostname === aReq.hostname) {
+      redirect = referer.path;
```

Comment:
Was there a reason why you chose to omit search query and hash e.g. `?yadda=ying#comment-someHash`?